polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -1,19 +1,13 @@
 import React, {useState, useCallback, useEffect} from 'react';
 
-import type {Experimental} from '../../types';
 import {classNames, variationName} from '../../utilities/css';
 import {useI18n} from '../../utilities/i18n';
 import {useIsAfterInitialMount} from '../../utilities/use-is-after-initial-mount';
 import {Image} from '../Image';
 
 import styles from './Avatar.scss';
 
-type Size =
-  | 'extraSmall'
-  | 'small'
-  | 'medium'
-  | 'large'
-  | Experimental<'xl' | '2xl'>;
+type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Do we not need `2xl` anymore? Looks like there are still mentions of it in the migration table and some other parts of this PR